MdeModulePkg/Core/Dxe/Mem/Page.c

```diff
@@ -616,16 +618,29 @@ CoreAddMemoryDescriptor (
     }
 
     if (gMemoryTypeInformation[Index].NumberOfPages != 0) {
+      // MU_CHANGE START Allow overriding of bin locations.
+      AllocationType = AllocateAnyPages;
+      CheckMemoryBinOverride (
+        Type,
+        &mMemoryTypeStatistics[Type].BaseAddress,

```

Comment:
is the base address necessary here? 

it gets passed into CheckMemoryBinOverride, but then in CoreAllocatepages below, it will be overwritten...

Reply:
Oh, Depending on the allocation type returned, it might be requesting a specific address.